loadpipeofficial/Hamza-medusa
# HAMSTR-27: Reconcile DBs

Updated setup to match Hamza-seller. 

To test: 
- run setup in Hamza-medusa (setup-0 and setup-1) 
- run Hamza-seller (server & client) 
- make sure that it runs properly
